Provider/src/main/java/com/spotify/confidence/EventSender.kt

```diff
@@ -6,11 +6,15 @@ interface EventSender : Contextual {
         payload: ConfidenceFieldsType = mapOf()
     )
 
+    fun track(eventProducer: EventProducer)
```

Comment:
Food for thought: should it be possible to untrack a producer?

Reply:
i thought about it but then it should probably be named or return an id or a handler to untrack etc. was overkill for a draft.